NewsFeed.Cmd.Tests/NewsFeed.Cmd.Tests.csproj

```diff
@@ -9,6 +9,10 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.6.1" />
     <PackageReference Include="Moq" Version="4.14.1" />
+    <PackageReference Include="StyleCop.Analyzers" Version="[1.1.118, )">
```

Comment:
Resolve build warning https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu1604